Providers/nxOMSAutomationWorker/automationworker/worker/httpclientfactory.py

```diff
@@ -32,13 +34,16 @@ def __init__(self, cert, key, insecure=False):
 
         # set proxy if valid proxy_configuration path is detected
         proxy_conf_path = configuration.get_proxy_configuration_path()
-        if configuration.get_proxy_configuration_path() != configuration.DEFAULT_PROXY_CONFIGURATION_PATH \
+        if proxy_conf_path != configuration.DEFAULT_PROXY_CONFIGURATION_PATH \
                 and os.path.isfile(proxy_conf_path):
-            proxy_conf_file = open(configuration.get_proxy_configuration_path(), "r")
-            proxy_configuration = proxy_conf_file.readline().strip()
-            if proxy_configuration != "":
-                self.proxy_configuration = proxy_configuration
-            proxy_conf_file.close()
+            if util.assert_file_read_permission(configuration.get_proxy_configuration_path()) is False:
```

Comment:
Oops  missed this in my refactoring. I'll fix on our repo for next release.
Thanks!